examples/tools/autotools/autotoolstoolchain/string_formatter/run_example.bat

```diff
@@ -0,0 +1,26 @@
+@echo ON
+
+set PROFILE_ARGS=-pr profiles/msys2
+set MSYSTEM=MINGW64
+set MSYS2_PATH_TYPE=inherit
+set CHERE_INVOKING=1
+
+
+echo "** INSTALL MSYS2 CONAN PACKAGE **"
+CALL conan install -r conancenter --requires=msys2/cci.latest -g VirtualBuildEnv
+if %ERRORLEVEL% neq 0 goto ERROR_EXIT
+
+
+echo "** ACTIVATE CONAN BUILD ENVIRONMENT **"
+CALL conanbuild.bat
+if %ERRORLEVEL% neq 0 goto ERROR_EXIT
+
+echo "** START MSYS2 BASH TERMINAL AND BUILD THE AUTOTOOLSTOOLCHAIN EXAMPLE **"
+CALL %MSYS_BIN%\bash.exe -c ./run_example.sh
```

Comment:
We need to invoke a bash from here, so we can execute autotools commands like automake and son on. The CI machine does not have a native bash installed, so let's use it directly from the msys2 conan package instead.